src/server/logging/log_format.js

```diff
@@ -111,30 +111,35 @@ export default class TransformObjStream extends Stream.Transform {
         'load'
       ]));
       data.message  = ansicolors.brightBlack('memory: ');
-      data.message += numeral(data.proc.mem.heapUsed).format('0.0b');
+      data.message += numeral(_.get(data, 'proc.mem.heapUsed')).format('0.0b');
       data.message += ' ';
       data.message += ansicolors.brightBlack('uptime: ');
-      data.message += numeral(data.proc.uptime).format('00:00:00');
+      data.message += numeral(_.get(data, 'proc.uptime')).format('00:00:00');
       data.message += ' ';
       data.message += ansicolors.brightBlack('load: [');
-      data.message += data.os.load.map(function (val) {
+      data.message += _.get(data, 'os.load', []).map(function (val) {
         return numeral(val).format('0.00');
       }).join(' ');
       data.message += ansicolors.brightBlack(']');
       data.message += ' ';
       data.message += ansicolors.brightBlack('delay: ');
-      data.message += numeral(data.proc.delay).format('0.000');
+      data.message += numeral(_.get(data, 'proc.delay')).format('0.000');
     }
     else if (data.type === 'error') {
       data.level = 'error';
-      data.message = event.error.message;
       data.error = serializeError(event.error);
       data.url = event.url;
+
+      const message = _.get(event, 'error.message');
+      data.message = (message && message !== '') ? message : `Event Error stack: ${_.get(event, 'error.stack')}`;
```

Comment:
nit, we don't need the `&& message !== ''` portion
```
> message = ''
''
> !!message
false
```

Reply:
we could use the _.get default arg as an alternative

Reply:
if `_.get()` results in an empty string, it won't pick the default arg though